File: ALittle/Module/ALittleIDE/Script/UIEdit/IDEEditCore/IDESetting/DisplayLayoutS.lua

module("ALittleIDE", package.seeall)

local ___rawset = rawset
local ___pairs = pairs
local ___ipairs = ipairs
local ___coroutine = coroutine


assert(DisplayObjectS, " extends class:DisplayObjectS is nil")
DisplayLayoutS = ALittle.Class(DisplayObjectS, "ALittleIDE.DisplayLayoutS")

function DisplayLayoutS:Ctor(user_info, tab_child, tree_logic)
	___rawset(self, "_layer_name", "ide_setting_displaylayout")
end

File: ALittle/Module/ALittleIDE/JSScript/UIEdit/IDEEditCore/IDESetting/DisplayLayoutS.js
{
if (typeof ALittleIDE === "undefined") ALittleIDE = {};


if (ALittleIDE.DisplayObjectS === undefined) throw new Error(" extends class:ALittleIDE.DisplayObjectS is undefined");
ALittleIDE.DisplayLayoutS = JavaScript.Class(ALittleIDE.DisplayObjectS, {
	Ctor : function(user_info, tab_child, tree_logic) {
		this._layer_name = "ide_setting_displaylayout";
	},
}, "ALittleIDE.DisplayLayoutS");

}